### PR TITLE
LPS-195012 | Poshi Automation for Bug | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Poshi Automation for Bug LPS-187080 Changing the category filter "Vocabulary" setting from "All" to "Filter" may not work properly.

Hi! :)
This pr is corresponding to this ticket: [LPS-187080](https://liferay.atlassian.net/browse/LPS-187080) → [LPS-195012](https://liferay.atlassian.net/browse/LPS-195012)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/Categories.testcase [hunk 1]
```diff
@@ -367,6 +367,55 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-187080. Changing the category filter 'Vocabulary' setting from 'All' to 'Filter' may not work properly."
+	@priority = 3
+	test CanConfigureCategoryFilter {
+		task ("Given a content page with a Category Filter widget") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/Categories.testcase:373]: Given a user has a Category Filter widget on a content page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/Categories.testcase [hunk 2]
```diff
@@ -367,6 +367,55 @@ definition {
...
+				vocabularyName = "Vocabulary Name");
+
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.addWidget(portletName = "Category Filter");
+		}
+
+		task ("When setting the Category Filter widget to Filter in the 'Vocabulary' setting") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/Categories.testcase:391]: When the user filters Category Filter by vocabularies

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @rafaellajordao, would you mind refactoring according to comments? Thanks

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase [hunk 3]
```diff
@@ -466,6 +466,55 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-187080. Changing the category filter 'Vocabulary' setting from 'All' to 'Filter' may not work properly."
+	@priority = 3
+	test CanConfigureCategoryFilter {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase:471]: Rename to FilterCategoryFilterByVocabularies

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @rafaellajordao, could you do this change? Thanks

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase (hunk 3 above)
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase]: done!

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**